frontend/src/features/recruit/components/recruitForm/RecruitListContainer.tsx

```
@@ -0,0 +1,104 @@
+            <button
+              className='relative h-5 w-5'
+              onClick={(e) => {
+                e.stopPropagation(); //부모의 클릭 이벤트(onItemClick) 방지
+                onEdit?.(doc.id);
+              }}
+            >
+              <I.EditIcon />
+            </button>
+            <button
+              className='relative h-5 w-5'
+              onClick={(e) => {
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)